docs/test-api/egison.json

```diff
@@ -0,0 +1,1226 @@
+[
```

Comment:
do we need all these json files in the `test-api` folder`?
those look like dumps from the API, but we should hit the API on page load, right?

Reply:
These are used during development so I would keep them there

Reply:
hmmm fair enough.
To me it feels a bit iffy to put this test data into the production output.
I think it's fine to have it in the source but after generating the production output I would expect it to be not there. But it's not super important.